src/main/webapp/js/ideas/ideas.factory.js

```diff
@@ -17,7 +17,8 @@
             removeIdea: removeIdea,
             changeIdeaLike: changeIdeaLike,
             getPage: getPage,
-            isUserAuthorOfIdea: isUserAuthorOfIdea
+            isUserAuthorOfIdea: isUserAuthorOfIdea,
+            getCommentsPageByIdeaId: getCommentsPageByIdeaId
```

Comment:
May be it makes sense to create new factory file for comments (if there will be more comment-related methods in the future)?